src/opm/output/eclipse/InteHEAD.cpp

```diff
@@ -912,16 +912,8 @@ namespace {
 
     int getMaximumAnalyticAquiferID(const Opm::AquiferConfig& cfg)
     {
-        auto maxAquID = [](const auto& aquiferCollection) -> int
-        {
-            return std::accumulate(aquiferCollection.begin(), aquiferCollection.end(), 0,
-                                   [](const int maxID, const auto& aquiferData)
-                                   {
-                                       return std::max(maxID, aquiferData.aquiferID);
-                                   });
-        };
-
-        return std::max(maxAquID(cfg.ct()), maxAquID(cfg.fetp()));
+        const auto& aquifer_ids = analyticAquiferIDs(cfg);
+        return *max_element(std::begin(aquifer_ids), std::end(aquifer_ids));
```

Comment:
This will dereference `aquifer_ids.end()` if `aquifer_ids.empty()`.  Could you not implement this some other way, e.g.
```C++
return std::max({ maxAquID(cfg.ct()), maxAquID(cfg.fetp()), maxAquiferID(cfg.constantFluxAquifers()) });
```

Reply:
AquiferFlux used std::unorder_map for the internal containers, it turns out to be a little more work.

Reply:
I will add an if guard here?